recipes/capnproto/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "0.5.3" %}
+
+package:
+  name: capnproto
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/sandstorm-io/capnproto/archive/v{{ version }}.tar.gz
+  md5: 909bd13ad6b8bc840ac78ab8f5bcb0a4
+
+build:
```

Comment:
For Windows, we have to do [this]( https://github.com/conda-forge/staged-recipes/wiki/VC-features ) at present. It's not optimal and we are trying to do away with it, but it is not gone yet. So, if we could add that stuff, that would be great.

Given `capnproto` requires C++11, we might need to add a `skip: true  # [win and py<35]` option here.